util/xrh_header.go

```diff
@@ -12,8 +12,7 @@ import (
 )
 
 const (
-	xrhAccountNumberKey string = "x-rh-sources-account-number"
-	xrhIdentityKey      string = "x-rh-identity"
+	xrhIdentityKey string = "x-rh-identity"
```

Comment:
I would suggest:

```diff
- xrhIdentityKey string = "x-rh-identity"
+ xRhIdentity string = "x-rh-identity"
```

Reply:
in this PR is solved constant related with account number and that is why here is removed only xrhAccountNumberKey and replaced by constant from middleware/headers/headers.go ... the xrhIdentityKey constant will be replaced here #579

Reply:
Oh, I see, thanks!